src/libs/actions/User.js

```diff
@@ -532,7 +532,18 @@ function subscribeToUserEvents() {
     // Handles the mega multipleEvents from Pusher which contains an array of single events.
     // Each single event is passed to PusherUtils in order to trigger the callbacks for that event
     PusherUtils.subscribeToPrivateUserChannelEvent(Pusher.TYPE.MULTIPLE_EVENTS, currentUserAccountID, (pushJSON) => {
-        _.each(pushJSON, (multipleEvent) => {
+        let updates;
+
+        // This is the old format where each update was just an array, with the new changes it's an object with
+        // lastUpdateID, previousUpdateID and updates
+        if (_.isArray(pushJSON)) {
+            updates = pushJSON;
+        } else {
+            // const lastUpdateID = pushJSON.lastUpdateID;
+            // const previousUpdateID = pushJSON.previousUpdateID;
```

Comment:
Was this code supposed to go in? 😄

Reply:
Yeah, it's just to show the person picking up the issue to implement this, to know the 2 pieces of data are here.